### PR TITLE
DGtal 0.9

### PR DESCRIPTION
**Pull request by dcoeurjo**
Upgrade DGtal to 0.9.
(sorry but I don't know how to manage bottle sha256 entries)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
The CI bot crashes on something which does not seem to be related to the formula..

File: dgtal.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Dgtal < Formula
   homepage "http://libdgtal.org"
```
Comment (dpo) [inline dgtal.rb]: It would be great if you could add a `desc` field with a (brief) description of the software.

Comment (dcoeurjo) [inline dgtal.rb]: oh sure..

**Comment by dpo** [comment]
Only the linux build failed. It hasn't been totally ironed out. All is well on OSX.

File: dgtal.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 class Dgtal < Formula
-  homepage "http://libdgtal.org"
-  url "http://liris.cnrs.fr/dgtal/releases/DGtal-0.8.0-Source.tar.gz"
-  sha1 "61c8d4b7db2c31daed9456ab65b0158d0a0e1bab"
+  desc "DGtal stands for Digital Geometry Tools and Algorithms Library. It is a toolkit to perform topology and geometry processing on (grid-based) digital data."
```
Comment (dpo) [inline dgtal.rb]: Could you please shorten this by removing "DGtal stands for"?

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dcoeurjo** [comment]
Description updated

**Comment by dpo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**